experiments/AMIP/modular/components/land/bucket_init.jl

```diff
@@ -162,6 +163,7 @@ function bucket_init(
     space,
     dt::FT,
     saveat::FT,
+    land_fraction,
```

Comment:
The named field in the struct was generic `area_fraction` (as for the other `SurfaceModelSimulation` types). Is this discrepancy here desired/intentional?

Reply:
Ahh, this was left from an old version. Well spotted!